### ISSUE TITLE
[FEATURE] Dynamic Namespace Selection while using embedded chat

### ISSUE THREAD

**Issue by marsalan06** [FlowiseAI/Flowise#2747]
Do we have a way where we can specify the name space for the content we want to fetch from the vector db while using embedded chat , i can do it in api request but then i wont be able to use the embedded chat , is there a way or am i missing something.

![image](https://github.com/FlowiseAI/Flowise/assets/50286252/7f019f5e-92bf-4f40-bc63-ad90679aba17)

**Comment by HenryHengZJ** [comment]
yep - you are looking for `chatflowConfig` - https://docs.flowiseai.com/using-flowise/embed#chatflow-config
